app/models/jurisdiction.rb

```diff
@@ -127,6 +128,14 @@ def unviewed_permit_applications
     permit_applications.unviewed
   end
 
+  def submission_inbox_set_up
+    PermitType.all.all? do |permit_type|
+      self.permit_type_submission_contacts.any? do |contact|
+        contact.permit_type_id == permit_type.id && contact.email.present? && contact.confirmed_at.present?
+      end
+    end
+  end
```

Comment:
can you enter this in chat GPT. I suspect there is a way to just do this in a single query using joins rather than looping through them all. Just a suggestion to explore